lib/collector/facts.js

```diff
@@ -37,12 +37,11 @@ function facts(agent, callback) {
       const environment = data.environment || []
 
       const packages = data.environment.find((elt) => elt[0] === 'Packages')[1]
-      const dedupPackages = new Set()
-      packages.forEach((dep) => dedupPackages.add(JSON.parse(dep)[0]))
-      LOG_LIBRARIES.forEach((lib) => {
-        if (dedupPackages.has(lib)) {
+      packages.forEach((pkg) => {
```

Comment:
i mentioned prob good to extract lines 39-47 into its own function and call it here so it's clear the intent of this snippet

Reply:
Oh, okay, done.